src/types/swiper-class.d.ts

```diff
@@ -444,10 +444,10 @@ interface Swiper extends SwiperClass<SwiperEvents> {
   keyboard: KeyboardMethods;
   lazy: LazyMethods;
   mousewheel: MousewheelMethods;
-  navigation: NavigationMethods;
-  pagination: PaginationMethods;
+  navigation: NavigationMethods | boolean;
```

Comment:
If someone uses TypeScript and used navigation methods in that way: `swiper.navigation.update();`. TypeScript will raise a error: "Property 'update' does not exist on type boolean | NavigationMethods"